### PR TITLE
Docs theme updates

### PR DESCRIPTION
**Pull request by jputrino**
### Proposed changes

This PR changes the configurations for the docs theme. We have moved the theme development to GitHub so it will be accessible in Netlify builds (and the _vendors directory is no longer required). The changes in this PR also let us manage future theme bumps by updating the NGINX_THEME_VERSION variable stored in the Netlify console.

The dev and staging sites are already updated with the changes. Following the merge of this PR, we will need to port the update to the release branch, too.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by lucacome — APPROVED** [review]
This is awesome! 🚀

**Review by ciarams87 — APPROVED** [review]
🤩 🚀

File: docs/Makefile [hunk 1]
```diff
@@ -1,40 +1,49 @@
...
+	if [[ -d ${PWD}/public ]] ; then rm -rf ${PWD}/public && echo "Removed public directory" ; else echo "Did not find a public directory to remove" ; fi
 
 hugo-mod:
-	hugo mod clean
-	rm -rf _vendor
-	hugo mod get
-	hugo mod vendor
+	hugo mod get $(THEME_MODULE)@v$(THEME_VERSION)
 
+# Builds using the Hugo "staging" environment
```
Comment (tomasohaodha) [inline docs/Makefile:20]: Typo - should be "production" environment

**Review by tomasohaodha — CHANGES_REQUESTED** [review]

File: docs/Makefile (hunk 1 above)
Comment (jputrino) [reply docs/Makefile:20]: good catch! fixed

**Review by tomasohaodha — APPROVED** [review]

**Pull request merged**